src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -38,6 +38,8 @@ def get(self, request, project):
             if hasattr(node, "form_fields"):
                 context["formFields"] = node.form_fields
 
+            if can_create_tickets and node.id in TICKET_ACTIONS:
```

Comment:
@ceorourke we don't need the `can_create_tickets` bit here since we'd already have skipped it if that were false.